wwwapp/views.py

```diff
@@ -588,3 +593,46 @@ def page(request):
 
 index_view = as_article("index")
 template_for_workshop_page_view = as_article("template_for_workshop_page")
+
+
+def resource_auth_view(request):
+    """
+    View checking permission for resource (header X-Original-URI). Returns 200
+    when currently logged in user should be granted access to resource and 403
+    when access should be denied.
+
+    See https://docs.nginx.com/nginx/admin-guide/security-controls/configuring-subrequest-authentication/
+    for intended usage.
```

Comment:
I'm still missing info on how exactly are the paths processed (which paths will be available when you give access to e.g. `/internety/www15`, and which paths are checked when you try to acess some URL [all prefixes])

Reply:
Such info is in the model docs.

Reply:
Such info is in the model.